src/board/leds.c

```diff
@@ -26,8 +25,8 @@
 /** Setup the LED GPIOs. */
 void led_setup(void)
 {
-  RCC_AHB1ENR |= RCC_AHB1ENR_IOPCEN;
-  gpio_mode_setup(GPIOC, GPIO_MODE_OUTPUT, GPIO_PUPD_NONE, GPIO3 | GPIO4);
+  palSetPadMode(GPIOC, GPIOC_LED1, PAL_MODE_OUTPUT_PUSHPULL);
```

Comment:
Not sure if it's in Chibi 2, but in 3 there is a GPIO line type which combines port and pad.